UKHomeOffice/cerberus-service
# COP-7634 Fix order of the activity logs on cerberus task details page

## Description
This PR makes the date prop on all activity elements the same format.
## To Test
- On main branch,  find target with activity log out of order i.e. `http://localhost:8080/tasks/AUTOTEST-RoRo-Tourist%2F104869%3ACMID%3DTEST`
- Pull PR branch and run natively
- Go to same target under same business key
- *It should be sorted correctly*
## Developer Checklist

\* Required
